src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Email/Blocks/email_configuration.html.twig

```diff
@@ -38,6 +38,19 @@
           <div class="js-smtp-configuration{% if emailConfigurationForm.mail_method.vars.value != smtpMailMethod %} d-none{% endif %}">
             {{ form_widget(emailConfigurationForm.smtp_config) }}
           </div>
+          {{ form_row(emailConfigurationForm.mail_type) }}
+          {{ form_row(emailConfigurationForm.log_emails) }}
+          <div class="alert alert-info" role="alert">
+            <p class="alert-text">
+              {{ 'Settings below allow you to configure DKIM signing of outgoing emails. This will greatly reduce the chance of your emails ending in SPAM folders.'|trans({}, 'Admin.Advparameters.Help') }}
```

Comment:
```suggestion
              {{ 'Fill in the fields below to set up DKIM signing of outgoing emails. This will reduce the likelihood of your emails being marked as spam.'|trans({}, 'Admin.Advparameters.Help') }}
```